sdk/src/main/java/com/hedera/hashgraph/sdk/TransactionResponse.java

```diff
@@ -29,6 +29,13 @@
         this.scheduledTransactionId = scheduledTransactionId;
     }
 
+    public TransactionReceipt getReceiptSync(Client client) throws Exception {
```

Comment:
Again, if we don't end up with an `executeSync` we should probably drop the `Sync` prefix on these for consistency.